packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -139,6 +139,24 @@ async def _resolve_location_id(
     return store_id
 
 
+async def complete_file_upload(
+    uploaded_parts: list[UploadedPart],
+    upload_completion_link: AnyUrl,
+    client_session: ClientSession | None = None,
+) -> ETag:
+    async with ClientSessionContextManager(client_session) as session:
+        e_tag: ETag | None = await _complete_upload(
+            session=session,
+            upload_completion_link=upload_completion_link,
+            parts=uploaded_parts,
+            is_directory=False,
+        )
+    assert (
+        e_tag is not None
+    )  # nosec - should be none because we are only uploading a file here

```

Comment:
the comment says the contrary to what you do here I think

Reply:
Fixed, thanks :smile: